tests/torch/ptq/test_helpers.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Looks like duplicate of `test_quantize_model_helpers.py`

Reply:
You are right, thanks!